specification/paths/ServiceGroups-service_group_id.json

```diff
@@ -0,0 +1,35 @@
+{
+  "get": {
+    "tags": [
+      "ServiceGroups"
+    ],
+    "summary": "Get a service group for a service contract by id.",
+    "description": "This endpoint retrieves a service group for the supplied service contract.",
```

Comment:
`for the supplied service contract` is this still the case? (also for the service-insurances and service-options endpoints)